### PR TITLE
Add a 'Create a new entity' action.

### PR DESCRIPTION
**Pull request by bojanz**
Implements a derivative plugin that provides a "Create a new $entity_type" for each content entity type.
Config entity types are not typed data aware yet, so they don't make sense in this context yet.

Disregard the summary() method, we said we want to kill it and provide it via the base class anyway.

Fixed the existing tests not to fail due to additional non-mocked method calls.

### CODE REVIEW

**Comment by bojanz** [comment]
The tests pass locally, but since there's no test introduced for the new functionality, this should be considered broken. My initial test attempt is up at https://gist.github.com/bojanz/963b31c5a930ecde0e6a

File: tests/src/Integration/RulesIntegrationTestBase.php [hunk 1]
```diff
@@ -137,6 +142,14 @@ public function setUp() {
       ->disableOriginalConstructor()
       ->getMock();
 
+    $this->entityManager = $this->getMockBuilder('Drupal\Core\Entity\EntityManagerInterface')
+      ->disableOriginalConstructor()
+      ->getMock();
+    $this->entityManager->expects($this->any())
+        ->method('getDefinitions')
```
Comment (fubhy) [inline tests/src/Integration/RulesIntegrationTestBase.php]: Indentation is wrong here :*(

File: tests/src/Integration/Action/FetchEntityByIdTest.php [hunk 2]
```diff
@@ -44,6 +44,12 @@ public function setUp() {
       ])
       ->getMock();
 
+    // The base field definitions for entity_test aren't used, and would
+    // require additional mocking.
+    $this->entityManager->expects($this->any())
+        ->method('getBaseFieldDefinitions')
```
Comment (fubhy) [inline tests/src/Integration/Action/FetchEntityByIdTest.php]: Indentation is wrong here :*(

Comment (bojanz) [inline tests/src/Integration/Action/FetchEntityByIdTest.php]: Fixed, thanks. Time to quit Sublime.

File: src/Plugin/Action/EntityCreate.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+   * @var \Drupal\Core\Entity\EntityManagerInterface
+   */
+  protected $entityManager;
+
+  /**
+   * The entity type.
+   *
+   * @var string
+   */
+  protected $entityType;
```
Comment (fago) [inline src/Plugin/Action/EntityCreate.php:39]: Should be $entityTypeId

**Comment by klausi** [comment]
continued in #158

**Pull request closed without merging**